src/Illuminate/Cache/TaggableStore.php

```diff
@@ -2,7 +2,9 @@
 
 namespace Illuminate\Cache;
 
-abstract class TaggableStore
+use Illuminate\Contracts\Cache\Store;
```

Comment:
Both `TaggedCache` and `TagSet` constuctors are typehinted with `Store` contract. Which means `TaggableStore` must implement it in order to work properly.
https://github.com/rkgrep/framework/blob/c163e00964207092bd627c7fc6fc46eb1c206c8e/src/Illuminate/Cache/TaggableStore.php#L17